### DISCUSSION TITLE
Deluxo mod for every vehicle

### DISCUSSION THREAD

**Discussion by AIdaveloper** [YimMenu/YimMenu#1207] [Ideas]
Add the ability to fly as Deluxo on every vehicle. There is the same thing in 0x and Nightfall menu.
Also mb possible toreador mod

**Comment by Tayipc38** [comment]
This sounds like a request instead of a discussion.

**Reply by AIdaveloper** [reply]
well, there is no branch with wishes, so I wrote this in the ideas branch
